task-launcher/src/tasks/shared/helpers/taskStore.js

```diff
@@ -62,7 +62,7 @@ export const setTaskStore = (config) => {
     // -----
     audioFeedback: config.audioFeedback,
     skipInstructions: config.skipInstructions,
-    corpusId: config.corpusId,
+    corpusId: config.corpusId ?? config.corpus,
```

Comment:
Where is the `corpus` value coming from? Is it from the actual corpus excel sheet? Can we simply rename the column to `corpusId` then?

Reply:
It is not coming from a column on the excel sheet, it the the actual name of the corpus we are pulling. I tried different ways to get it, and I actually think we don't need `corpusId`, just `corpus`

Reply:
Okay got it. In that case, please add a new variable called `corpus` and just assign the corpus to that instead of assigning it to the corpusId.

Reply:
Sure!